README.md

```diff
@@ -151,6 +151,24 @@ UnSubscribing from channel messages
 await channelMessageSubscription.cancel();
 ```
 
+Publishing channel messages
+
+```dart
+//publish name and data
```

Comment:
Please always include a space after `//` comment openers. e.g., in this case:

```suggestion
// publish name and data
```

Reply:
Applied this suggestion on all comments

e8eab0f6e60e99c4e8811943ab2b7a801e064ce4